config/secrets.yml

```diff
@@ -1,7 +1,5 @@
 default: &default
   secret_key_base: MGhvYnpIWnJaRHhLaDJrRkpOd1JqWFhHUHMuRFpVeGRhRE5GVHY1NDdBQkwxM0NkWGdnY1NnZmhNMWhCRzBBRG80U2YuTG5hMURrb3NkZ2VURkJOelZkZmhFMUM0TWJwdGFwcFJhM21hRHdQVTFoRUpWZXZCbmp3YnROZkZ0a2MK
-  http_auth_username: <%= ENV["SP_NAME"] %>
```

Comment:
Removing it from here will break the dashboard deploy status checker, we use these secrets to load the status deploy JSON files in the status checker: https://github.com/18F/identity-dashboard/blob/master/app/services/deploy_status_checker.rb#L89